src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/PublicationService.cs

```diff
@@ -0,0 +1,61 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Api.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Services;
+
+internal class PublicationService : IPublicationService
+{
+    private readonly PublicDataDbContext _publicDataDbContext;
+    private readonly IContentApiClient _contentApiClient;
+
+    public PublicationService(PublicDataDbContext publicDataDbContext, IContentApiClient contentApiClient)
+    {
+        this._publicDataDbContext = publicDataDbContext;
+        this._contentApiClient = contentApiClient;
+    }
+
+    public async Task<Either<ActionResult, PaginatedPublicationListViewModel>> ListPublications(
+        int page,
+        int pageSize, 
+        string? search = null)
+    {
+        return await GetPublishedDataSetPublicationIds(_publicDataDbContext)
```

Comment:
Adding a comment for myself here. Don't need to pass `_publicDataDbContext` into this method